tasks/main.yml

```diff
@@ -8,6 +8,19 @@
     redis_package: "{{ __redis_package }}"
   when: redis_package is not defined
 
+- name: Ensure Redis configuration dir exists.
+  file:
+    path: "{{ redis_conf_path | dirname }}"
+    state: directory
+    mode: 0755
+
+- name: Ensure Redis is configured.
+  template:
+    src: redis.conf.j2
```

Comment:
Wouldn't putting this in place prior to Redis' installation cause Redis to overwrite the file?

Reply:
Nope, it works as expected.

Dpkg leaves existing files alone according to docs:
https://www.debian.org/doc/debian-policy/ap-pkg-conffiles.html

Same logic for rpm-based distros:
https://docs.fedoraproject.org/ro/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch09s05s03.html

Reply:
And same for Arch:
https://wiki.archlinux.org/index.php/Pacman#What_happens_during_package_install/upgrade/removal

Reply:
@geerlingguy I can confirm that Redis keeps the existing config that was in place before the install.